config/routes.rb

```diff
@@ -123,7 +123,12 @@
     resource :search,    only: :show
     resource :dashboard, only: :show, constraints: { format: /html|atom/ }
     resources :profiles, only: :show
-    resource :profile, only: [:edit, :update]
+    resource :profile, only: [:edit, :update] do
+      member do
+        get :delete
+        delete :destroy, as: :destroy
```

Comment:
If you change _this_ `resource :profile, only: [:edit, :update]` to `resource :profile, only: [:edit, :update, :destroy]` it will work the way we want.